CHANGELOG.md

```diff
@@ -6,27 +6,18 @@
 
 ## Internal
 - Add `syncGroup` property to the `Facility` resource
-- Show medicines required error in `TeleconsultPrescriptionScreen`
-- Remove `TeleconsultationApi` & load medical officers from `MedicalOfficer` table
 - Remove custom WorkManager initializer
 - Bump Android Gradle Plugin version to 4.0.2
 - Add method to `AppDatabase` to clear all patient data not in the current sync group
-- Request storage permission when downloading or sharing prescription
-- Fix small UI inconsistencies in share prescription screen
-- Hide keyboard when exiting from `TeleconsultPrescriptionScreen`
-- Add file provider for sharing prescription image below Android 10
-
-## Features
-- Add `TeleconsultSharePrescriptionScreen` 
-- Fix prescribed drug frequency type adapter
 
 ## On Demo
```

Comment:
Should we be updating `On Demo` to `releasing to PROD date and version` as well? Since we're releasing it to `PROD` ?

Reply:
That will be done later, I haven't made the PROD release yet.